metaflow/cmd/configure_cmd.py

```diff
@@ -248,18 +248,6 @@ def configure_s3_datastore(existing_env):
         default=existing_env.get("METAFLOW_DATASTORE_SYSROOT_S3"),
         show_default=True,
     )
-    # Set Amazon S3 folder for datatools.
-    env["METAFLOW_DATATOOLS_SYSROOT_S3"] = click.prompt(
```

Comment:
Had a quick chat about this today and as it seems that the `METAFLOW_DATATOOLS_S3ROOT` has been used by `metaflow_config.py` since the initial commit, let's line up the aws configuration with this instead of removing the config option altogether.

Reply:
Sure, makes sense. 

To clarify, you want to keep `METAFLOW_DATATOOLS_S3ROOT` and update `metaflow_config.py` to use it? Or you want to add `DATATOOLS_SUFFIX`, which is currently used in `metaflow_config.py`, to the `metaflow configure aws` command?

Reply:
instead of removing the `METAFLOW_DATATOOLS_SYSROOT_S3` prompt from `configure_cmd.py`, I'm suggesting to instead rename the variable to `METAFLOW_DATATOOLS_S3ROOT` which is what `metaflow_config.py` uses, and which seems to have been the intention of the config prompt as well

Reply:
ok, cool. Makes sense. 

I'll make the update.

Reply:
looks great, this should be good to go :)

thanks for working on this 👍